packages/fast-components-styles-msft/src/design-system/index.ts

```diff
@@ -2,12 +2,14 @@ export interface IDesignSystem {
     foregroundColor: string;
     backgroundColor: string;
     brandColor: string;
+    gray: string;

```

Comment:
We were going to pull this as a global, right? If we need tints and shades, we should consider holding those values in an object as part of the design system:
```
const shades = {
    shade10 = '#######'
}
```

in design system we can reference the objects.

Reply:
Oh I meant to pull - I am creating shades with mixtures - updating